v3/src/components/graph/adornments/univariate-measures/mean-absolute-deviation/mean-absolute-deviation-adornment-model.ts

```diff
@@ -10,6 +10,7 @@ export const MeanAbsoluteDeviationAdornmentModel = UnivariateMeasureAdornmentMod
   .named("MeanAbsoluteDeviationAdornmentModel")
   .props({
     type: "Mean Absolute Deviation",
+    hasRange: true,
```

Comment:
Make this a view that returns `true` rather than a property -- there's no reason to store it in the model/document given that it never changes.

Reply:
Ah, OK 👍